TSError when building block editor tools

Since #13 was merged, trying to build the package returns the following TypeScript error:

```
../../node_modules/@types/node/globals.d.ts(72,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'AbortSignal' must be of type '{ new (): AbortSignal; prototype: AbortSignal; abort(reason?: any): AbortSignal; timeout(milliseconds: number): AbortSignal; }', but here has type '{ new (): AbortSignal; prototype: AbortSignal; }'.
```

--- a/packages/block-editor-tools/package.json
+++ b/packages/block-editor-tools/package.json
@@ -15,9 +15,9 @@
   },
   "scripts": {
     "prebuild-types": "check-node-version --package",
-    "build-types": "npx -p typescript tsc src/index.js src/**/*.js --declaration --allowJs --emitDeclarationOnly --outDir build",
+    "build-types": "npx tsc --declaration --emitDeclarationOnly",
     "prepublishOnly": "npm run build",
-    "prebuild": "check-node-version --package",
+    "prebuild": "check-node-version --package && rm -rf build/*",
     "build": "webpack --mode=production && npm run build-types",
     "predev": "check-node-version --package",
     "dev": "webpack --mode=development --watch",
@@ -62,7 +62,7 @@
     "sass": "^1.53.0",
     "sass-loader": "^13.0.2",
     "style-loader": "^3.3.1",
-    "typescript": "^4.7.4",
+    "typescript": "^4.9.3",
     "webpack": "^5.73.0",
     "webpack-cli": "^4.10.0"
   }

--- a/packages/block-editor-tools/tsconfig.json
+++ b/packages/block-editor-tools/tsconfig.json
@@ -9,18 +9,21 @@
     "module": "es6",
     "moduleResolution": "node",
     "noImplicitAny": true,
-    "outDir": "./dist/",
+    "outDir": "./build/",
     "paths": {
       "@/*": ["*"]
     },
     "sourceMap": true,
+    "skipLibCheck": true,
     "target": "es5"
   },
   "exclude": [
     "jest.config.js",
     "lib",
+    "build",
     "node_modules",
     "tests",
-    "vendor"
+    "vendor",
+    "webpack.config.js"
   ]
 }

